src/main/java/com/bettercloud/kadmin/services/BasicKafkaConsumerProviderService.java

```diff
@@ -40,13 +47,32 @@
     @Autowired
     public BasicKafkaConsumerProviderService(
             @Value("${kafka.host:localhost:9092}")
-            String defaultKafkaHost,
+                    String defaultKafkaHost,
             @Value("${schema.registry.url:http://localhost:8081}")
-            String defaultSchemaRegistryUrl,
+                    String defaultSchemaRegistryUrl,
+            @Value("${security.protocol:PLAINTEXT}")
+                    String defaultSecurityProtocol,
+            @Value("${trust.store.location}")
+                    String defaultTrustStoreLocation,
+            @Value("${trust.store.password}")
+                    String defaultTrustStorePassword,
+            @Value("${key.store.location}")
+                    String defaultKeyStoreLocation,
+            @Value("${key.store.password}")
+                    String defaultKeyStorePassword,
+            @Value("${key.password}")
+                    String defaultKeyPassword,
```

Comment:
This is up to personal style, but when you have this many `@Value` injected properties, it is usually easier to just pull the values out of the Spring `Environment` bean. But this approach is fine.